mapbox_vector_tile/encoder.py

```diff
@@ -229,6 +229,11 @@ def _get_feature_type(self, shape):
             return self.tile.LineString
         elif shape.type == 'Polygon' or shape.type == 'MultiPolygon':
             return self.tile.Polygon
+        elif shape.type == 'GeometryCollection':
+            raise ValueError('Encoding geometry collections not supported')
+        else:
+            raise ValueError('Cannot encode unknown geometry type: %s' %
+                             shape.type)
```

Comment:
I think this is dead code as shapely only supports:
Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon

I guess that's why you can't create a test for it also.